app/src/main/res/layout-sw600dp-port/profile_chooser_add_view.xml

```diff
@@ -16,14 +16,12 @@
       type="androidx.databinding.ObservableField&lt;Boolean&gt;" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
@Uticodes please format code of all xml layouts using shortcut **Windows**: `Ctrl + Alt + L` 
**macOS**: `Option + Command + L` **Linux**: `Ctrl + Shift + Alt + L`

Reply:
@MohitGupta121 strokeColor and formatting fixed